microshift_running_apps/microshift-deleting-resource-manifests.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-deleting-resource-manifests"]
+= Deleting resource manifests
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-deleting-resource-manifests
+
+toc::[]
+
+{microshift-short} supports the deletion of resource manifests for the following use cases:
+
+* Data removal - when you need to completely remove a resource from the cluster.
+* Upgrade - during an upgrade, some resources may need to be removed while others are retained to preserve data.
```

Comment:
I would keep it close to manifest deletion, not venture into microshift's upgrade, or data removal

```suggestion
* Manifest removal - when you need to completely remove a resource from the cluster.
* Manifest upgrade - during an application's upgrade, some resources may need to be removed while others are retained to preserve data.
```

Reply:
done